data_diff/sql.py

```diff
@@ -80,6 +80,12 @@ def compile(self, parent_c: Compiler):
         if self.where:
             select += " WHERE " + " AND ".join(map(c.compile, self.where))
 
+        if self.where_or:
```

Comment:
What's this?

Reply:
Ah yeah this is a relic left over from the first iteration where I found checkpoints still with this method, will remove